cloudformation/build-bundle-test-workflow.yml

```diff
@@ -0,0 +1,59 @@
+Description: Template for setting up resources for the build-bundle-test workflows
+Resources:
+  BuildBucket:
+    Type: "AWS::S3::Bucket"
+  BuildUser:
+    Type: "AWS::IAM::User"
+    Properties:
+      UserName: builder
+      Policies:
+        - PolicyName: S3_Write_Access
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              Effect: Allow
+              Action: "s3:PutObject"
+              Resource:
+                - !Sub "arn:aws:s3:::${BuildBucket}/builds/*"
+  BundleUser:
+    Type: "AWS::IAM::User"
+    Properties:
+      UserName: bundler
+      Policies:
+        - PolicyName: S3_Build_Read_Access
```

Comment:
Aren't these already public buckets - is granting read access needed?

Reply:
Only the prod bucket will be public. For our beta stack and individual developer stacks everything should be private. Also, IAM roles still need permissions to invoke GetObject (S3's permissions model is more complicated than other AWS services...access is gated by both the bucket policy and the caller's permissions).